src/NuGet.Core/NuGet.Packaging/Signing/Signatures/Signature.cs

```diff
@@ -31,6 +31,16 @@ public class Signature
         /// </summary>
         public SignatureContent SignatureContent { get; }
 
+        /// <summary>
+        /// Signature timestamp.
+        /// </summary>
+        public DateTimeOffset? Timestamp { get; }
+
+        /// <summary>
+        /// SignerInfo for the timestamp
```

Comment:
nit: missing  period at the end of the sentence.

Reply:
np